actors/market/tests/cron_tick_timedout_deals.rs

```diff
@@ -85,7 +86,7 @@ fn publishing_timed_out_deal_again_should_work_after_cron_tick_as_it_should_no_l
     let client_deal_proposal =
         ClientDealProposal { proposal: deal_proposal2.clone(), client_signature: sig };
     let params = PublishStorageDealsParams { deals: vec![client_deal_proposal] };
-    rt.expect_validate_caller_type(vec![*ACCOUNT_ACTOR_CODE_ID, *MULTISIG_ACTOR_CODE_ID]);
+    rt.expect_validate_caller_type(vec![Type::Account, Type::Multisig]);
```

Comment:
Since you're changing these lines, please change this and all the other cases where these two types are passed to use `CALLER_TYPES_SIGNABLE` (the one from `shared`) instead.